manifests/overlays/prod/configs/argo_cm/resource.inclusions

```diff
@@ -6,6 +6,7 @@
   - https://api.thoth01.lab.upshift.rdu2.redhat.com:6443
   - https://api.thoth12.lab.upshift.rdu2.redhat.com:6443
   - https://api.ocp4.idh.dev.datahub.redhat.com:6443
+  - https://api.rhods-idh.dev.datahub.redhat.com
```

Comment:
Looking at the other cluster URLs the port may be missing here

Reply:
Thanks for pointing that. Let me add the port